src/Illuminate/Database/Capsule/Manager.php

```diff
@@ -81,7 +81,7 @@ public static function connection($connection = null)
      * @param  string|null  $connection
      * @return \Illuminate\Database\Query\Builder
      */
-    public static function table($table, $connection = null)
+    public static function table($table, $as = null, $connection = null)
```

Comment:
What is the `$as` parameter here for? I can't see it being used on the static method.

Reply:
Why not? If this is the entry point into the query, where you specify the table, you may want to specify a subquery.

Reply:
Sorry, I meant that the method definition is still

```php
return static::$instance->connection($connection)->table($table);
```

Should it be passing it like this instead?

```php
return static::$instance->connection($connection)->table($table, $as);
```

Reply:
🤦‍♂ 

Fixing.

Reply:
Fixed. Thanks @georgehanson!

Reply:
No problem, I'm not sure why the tests didn't catch that error though...

Might want to update the docblock too :)

Reply:
> No problem, I'm not sure why the tests didn't catch that error though...

😬 

> Might want to update the docblock too :)

Ugh, I need more coffee. Thanks again, will do.